integration/kubernetes/init.sh

```diff
@@ -18,7 +18,11 @@ set -e
 SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
 source "${SCRIPT_PATH}/lib.sh"
 
-sudo -E kubeadm init --pod-network-cidr 10.244.0.0/16
+# Run init ignoring CRI preflight check.
+# the CRI preflight check verifies that /var/run/dockershim.sock exists. But we don't
+# need it as we use CRI-O for launching pods, then we need to disable this check,
+# otherwise, kubeadm will fail.
+sudo -E kubeadm init --pod-network-cidr 10.244.0.0/16 --ignore-preflight-errors=CRI
```

Comment:
I would add ```--cri-socket=/var/run/crio/crio.sock``` to the init and the reset.  This tells kubeadm to use that crio socket instead of docker to reset the system.  Without this (at least on the reset), you'll need to do painful cleanup (as we have to do today) after tearing down the cluster.

Reply:
Agree @egernst @jcvenegas, changes applied.